client/src/modules/list/components/ItemsContainer.scss

```diff
@@ -146,4 +143,69 @@
     font-size: 12px;
     line-height: 1.2;
   }
+
+  &__filter {
```

Comment:
why this style is here and not in `Filter.scss`?

Reply:
I've been thinking about adding this style to a separated file but @alemikolo said that we may want to have different styles for this component in different places.

@alemikolo  can you refer to that?

Reply:
I thought that this component the component will only have the functionality implemented. Because we may want to style it in a different way in different places.

We use it to filter lists in `move the item to list`  panel, now its added to `itemsList` to filter items.  Łukasz said that he would like to have such kind of filter in activities panel.

In our desktop view in all those places, the filter should have at least slightly different styles...

Maybe it would be better to put filter styles in a separate file, and overwrite them only when it is necessary?

Reply:
Move to component css file all default styles and override in other places elements that needs that.